### PR TITLE
docsify: add page

### PR DESCRIPTION
**Pull request by boldandbrad**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 4.x

### CODE REVIEW

**Comment by boldandbrad** [comment]
Addresses Issue: https://github.com/tldr-pages/tldr/issues/8184

**Comment by boldandbrad** [comment]
Fixed check failures!

File: pages/common/docsify.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Initialize new documentation in the current directory:
+
+`docsify init`
+
+- Generate new documentation in the specified directory:
+
+`docsify init {{path}}`
+
+- Serve local documentation on localhost:3000 with live reload:
```
Comment (marchersimon) [inline pages/common/docsify.md:14]:
```suggestion
- Serve local documentation on `localhost:3000` with live reload:
```

File: pages/common/docsify.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# docsify
+
+> Initialize and serve markdown documentation.
+> More information: <https://cli.docsifyjs.org>.
+
+- Initialize new documentation in the current directory:
```
Comment (marchersimon) [inline pages/common/docsify.md:6]: Not entirely sure, but I think this sounds a little more complete. Same for the second example
```suggestion
- Initialize a new documentation in the current directory:
```

**Review by marchersimon — COMMENTED** [review]
Just two minor suggestions from me.

Would you maybe also like to document some more examples like `--theme`, `generate`, or `--plugins`? They sound useful

File: pages/common/docsify.md (hunk 2 above)
Comment (boldandbrad) [reply pages/common/docsify.md]: Adopted.

**Comment by boldandbrad** [comment]
Added a couple more examples and committed suggested changes.

File: pages/common/docsify.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`docsify serve {{path}}`
+
+- Serve local documentation on `localhost` at the specified port:
+
+`docsify serve -p {{port}} {{path}}`
+
+- Generate a sidebar markdown file in the specified directory:
+
+`docsify generate {{path}}`
```
Comment (marchersimon) [inline pages/common/docsify.md:24]:
```suggestion
`docsify init {{path/to/directory}}`

- Serve local documentation on `localhost:3000` with live reload:

`docsify serve {{path/to/directory}}`

- Serve local documentation on `localhost` at the specified port:

`docsify serve --port {{port}} {{path/to/directory}}`

- Generate a sidebar markdown file in the specified directory:

`docsify generate {{path/to/directory}}`
```
(Just tldr conventions)

**Review by marchersimon — APPROVED** [review]
One last change. Looks good otherwise, thanks!

**Comment by boldandbrad** [comment]
Adopted suggestion!

File: pages/common/docsify.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`docsify init {{path/to/directory}}`
+
+- Serve local documentation on `localhost:3000` with live reload:
+
+`docsify serve {{path/to/directory}}`
+
+- Serve local documentation on `localhost` at the specified port:
+
+`docsify serve --port {{port}} {{path/to/directory}}`
```
Comment (CleanMachine1) [inline pages/common/docsify.md:20]:
```suggestion
`docsify serve --port {{80}} {{path/to/directory}}`
```

Feel free to ignore this one if it is incorrect, however I feel this makes it a little easier for readers

**Review by CleanMachine1 — APPROVED** [review]
LGTM after my suggestion is reviewed

Thanks for your contribution!

**Comment by boldandbrad** [comment]
Thanks for the reviews! I believe it is good to go!

**Comment by boldandbrad** [comment]
@CleanMachine1 are you able to merge? Thanks!

**Comment by marchersimon** [comment]
@boldandbrad we usually leave PRs open for at least a day to give everyone a chance to review it.

**Comment by boldandbrad** [comment]
Good to know! Thanks!

**Pull request merged**